entrypoint.sh

```diff
@@ -392,6 +401,24 @@ then
             helm cm-push --force ".tmp/charts/${chart_name}-${INPUT_CHART_VERSION}.tgz" repo
             ;;
 
+        helm-publish-oci)
+            is_set INPUT_CHART_APP_VERSION
+            is_set INPUT_CHART_PATH
+            is_set INPUT_CHART_VERSION
+            is_set INPUT_CHART_REPO
+            is_set INPUT_CHART_REPO_PASSWORD
+            is_set INPUT_CHART_REPO_USERNAME
+
+            helm_package
+
+            echo "-- Login OCI registry ${INPUT_CHART_REPO}, as ${INPUT_CHART_REPO_USERNAME}"
+            echo "${INPUT_CHART_REPO_PASSWORD}" | helm registry login ${INPUT_CHART_REPO} --username ${INPUT_CHART_REPO_USERNAME} --password-stdin
+
+            echo "-- Push OCI chart"
+            chart_name=$(basename "${INPUT_CHART_PATH}")
+            helm push ".tmp/charts/${chart_name}-${INPUT_CHART_VERSION}.tgz" oci://"${INPUT_CHART_REPO}"
```

Comment:
Super tiny nit: wrap the whole string in double quotes instead of just the variable.